src_c/_freetype.c

```diff
@@ -2046,8 +2046,21 @@ _ft_get_version(PyObject *self, PyObject *args, PyObject *kwargs)
     }
 
     if (linked) {
```

Comment:
Just coming back to this PR, sorry it's been a while.

I'd like you to add a comment in the code explaining why you're instantiating a whole freetype library here. I believe the answer is so you can query version independently of the pygame-freetype module being initialized.

I just think it could be helpful to someone in the future wondering why this is written this way.